bot/resource/cog/association/embed/asetin_embed.py

```diff
@@ -13,21 +13,21 @@ class AsetinEmbed:
 
     WHO_WE_ARE_FIELD_NAME: str = "Qui sommes-nous?"
     WHO_WE_ARE_FIELD_DESCRIPTION: str = (
-        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial."
+        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial. L'ASETIN représente les intérêts des étudiants et étudiantes de premier cycle en informatique auprès des différentes instances officielles de l’Université Laval, sur les différents comités et devant les autres associations étudiantes."
     )
 
     PART_OF_FIELD_NAME: str = "Qui en fait partie?"
-    PART_OF_FIELD_VALUE: str = "IFT, GLO, IGG et certificat"
+    PART_OF_FIELD_VALUE: str = "Les étudiants en informatique, génie logiciel, informatique et gestion ainsi que les étudiants inscrits aux certificats en informatique"
```

Comment:
Maybe rework this line as a list. See below the result:
![image](https://github.com/user-attachments/assets/1c4864cc-00e9-4d56-bebe-e39b505933d1)

Reply:
Will do